styles.css

```diff
@@ -384,3 +384,76 @@ nav li a img {
   width: 50px;
   height: 50px;
 }
+
+.contact {
+  background-image: url('contactshapes.png'), url('contact.png');
```

Comment:
- [ ] While you did a good job styling your contact form :+1:, to match with the Figma template I think it would be better if you add the required background as you can see below.

**Figma**

![Screenshot from 2023-01-24 22-51-34](https://user-images.githubusercontent.com/105117832/214476013-6bcd8a97-8bea-440d-a576-82d209e5f1f6.png)

**Your design**

![Screenshot from 2023-01-24 22-49-34](https://user-images.githubusercontent.com/105117832/214476047-27b38186-fcdd-4b6a-98c7-abbd0003365b.png)

- I have noticed that you are calling the background 'contact.png' instead of using 'Contact.png'

Reply:
Can you please guide a little further on this. As I think I have made the design similar to Figma template and it is completely resembling the Figma style 
![image](https://user-images.githubusercontent.com/117091923/214478866-f9e8d658-11b7-46c1-8af2-393ce2d63ca8.png)

Reply:
Do you mean that I need to use text instead of placeholder text in Input type= text filed and email field ?? Can you please kindly confirm on this ??

Reply:
Hi @baqar-abbas, no I mean that the background on your last section is not showing. Maybe it is working on your local machine, but when I open it, it is not working

Reply:
One reason could be that your background image is capitalized

Reply:
The one that you save as “Contact.png”

Reply:
This one 
![BCB9DC22-849A-4B6C-800F-FD175AAA88E3](https://user-images.githubusercontent.com/105117832/214480652-aace879c-6846-4564-936f-da86a983be66.jpeg)

and when you call it in your css you are using “contact.png” instead of “Contact.png”

Reply:
Can I Submit my Review again for review please comment on that ?? @IngDavidTM . For me the design appears to be similar as Figma Desgin Do I need to add Text in a text field instead of "Enter your name here"(Placeholder text)
? Please confirm on that ??

Reply:
Of course you can 👍 and it is not necessary to add the text “Yeremias” instead the one that you are using

Reply:
![image](https://user-images.githubusercontent.com/117091923/214489672-7f3c7c4d-0a37-4082-b57f-f9dbfea0d182.png)
It is actually not a wide image width of 375px is used here. I don't know why it is not appearing well at your side but it is working perfectly fine on my machine. Ok I am submitting it again for your kind review.

Reply:
Made the required changes and submitting it for re-review